draft-ietf-trans-rfc6962-bis.md

```diff
@@ -212,6 +212,33 @@ interpreted as described in RFC 2119 [RFC2119].
 Data structures are defined according to the conventions laid out in Section 4
 of [RFC5246].
 
+## Major Differences from RFC6962
```

Comment:
I wouldn't mind mentioning it.
However, as you point out, that's not a protocol change per-se, so IMHO it may be less of an interest to whoever reads this section.